### PR TITLE
[quant] Make OP_LIST_TO_FUSER_METHOD public

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43377 [quant][graphmode][fx] Add support for instance_norm
* #43376 [quant][graphmode][fx] Add support for layer_norm
* #43375 [quant][graphmode][fx] Add support for elu
* #43374 [quant][graphmode][fx] Add support for hardswish
* #43373 [quant][graphmode][fx] Add support for mul and mul relu
* #43372 [quant][grapphmode][fx][test][refactor] Refactor quantized add test
* #43335 [quant][graphmode][fx] Add support for batchnorm relu
* #43334 [quant][graphmode][fx] Add support for batchnorm
* #43333 [quant][graphmode][fx] Add support for cat
* #43332 [quant][graphmode][fx] Add support for add relu
* #43331 [quant][graphmode][fx] Add support for add
* #43287 [quant][graphmode][fx] Add support for conv module + relu
* **#43286 [quant] Make OP_LIST_TO_FUSER_METHOD public**
* #43285 [quant][graphmode][fx] Add support for conv module

Summary:
We need to use this in graph mode quantization on fx

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23221734](https://our.internmc.facebook.com/intern/diff/D23221734)

### CODE REVIEW

**Review by vkuzo — APPROVED** [review]
lg.  Optional suggeston for future PRs: would it make sense to have a clean split between:
* eager mode
* fx
* shared between eager mode and fx

i.e. the shared code would be in a separate directory or file, to make it clear what's shared and what's not, with ideally the shared part of the codebase growing over time, and eager-only shrinking?

**Comment by jerryzh168** [comment]
> lg. Optional suggeston for future PRs: would it make sense to have a clean split between:
> 
> * eager mode
> * fx
> * shared between eager mode and fx
> 
> i.e. the shared code would be in a separate directory or file, to make it clear what's shared and what's not, with ideally the shared part of the codebase growing over time, and eager-only shrinking?

If you just talk about things under torch/quantization then there is not much that is eager mode only, probably just https://github.com/pytorch/pytorch/blob/master/torch/quantization/quantize.py and https://github.com/pytorch/pytorch/blob/master/torch/quantization/stubs.py I think. There is also https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/functional_modules.py which is only used in eager mode.

we can certainly put them in the same folder, but not sure if it's worth it or not.

**Review by raghuramank100 — APPROVED** [review]
